Dockerfile

```diff
@@ -19,8 +19,12 @@ COPY ./ /harvester
 # REST service app requires mod_rewrite.
 RUN a2enmod rewrite && \
     # Composer for building Harvester.
-    curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer && \
+    curl -o /tmp/composer-installer https://getcomposer.org/installer && \
+    curl -o /tmp/composer-installer.sig https://composer.github.io/installer.sig &&  \
+    php -r "if (hash('SHA384', file_get_contents('/tmp/composer-installer')) !== trim(file_get_contents('/tmp/composer-installer.sig'))) { unlink('/tmp/composer-installer'); echo 'Invalid installer' . PHP_EOL; exit(1); }" && \
+    php /tmp/composer-installer --version=1.6.5 --filename=composer --install-dir=/usr/local/bin && \
+    php -r "unlink('/tmp/composer-installer');" && \
+    php -r "unlink('/tmp/composer-installer.sig');" && \
```

Comment:
Can we simplify this with multistage builds using the official Docker image?

Also: Could you please add a note why we want to use Composer 1.6.5? Did you go through a process of testing versions, compared Composer release dates with commit dates to try to determine which version was used the last or just pick a random old version?